client/apps/edit/components/section_callout/index.coffee

```diff
@@ -0,0 +1,188 @@
+#
+# Callout section that only appears between two text sections. It can be used
+# to add other articles, or to use as a pull quote
+#
+
+Backbone = require 'backbone'
+_ = require 'underscore'
+React = require 'react'
+sd = require('sharify').data
+gemup = require 'gemup'
+Autocomplete = require '../../../../components/autocomplete/index.coffee'
+{ div, section, input, a, h1, h2, button, img, p, strong, span } = React.DOM
+{ crop } = require('embedly-view-helpers')(sd.EMBEDLY_KEY)
+Article = require '../../../../models/article.coffee'
+
+module.exports = React.createClass
```

Comment:
:clap: Well done with this component! It's like the dream team of components—`crop` `gemup` `Autocomplete` and tons of functionality of it's own.